packages/dx-react-scheduler-demos/src/demo-sources/scheduler-editing/material-ui/drag-drop.jsx

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable react/destructuring-assignment */
+import * as React from 'react';
+import Paper from '@material-ui/core/Paper';
+import { ViewState, EditingState } from '@devexpress/dx-react-scheduler';
+import {
+  Scheduler,
+  WeekView,
+  Appointments,
+  DragDropProvider,
+} from '@devexpress/dx-react-scheduler-material-ui';
+import { appointments } from '../../../demo-data/appointments';
+
+const dragDisableIds = [3, 8, 10, 12];
```

Comment:
[Set](https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Set) is better for *contains* checks.